tests/run-make/translation/rmake.rs

```diff
@@ -0,0 +1,89 @@
+// Various tests on Fluent bundles, useful to change the compiler's language
+// to the one requested by the user. Check each comment header to learn the purpose
+// of each test case.
+// See https://github.com/rust-lang/rust/pull/95512
+
+//@ needs-symlink
+
+use run_make_support::{path, rfs, rustc};
+
+fn main() {
+    // Check that the test works normally, using the built-in fallback bundle.
+    rustc().input("test.rs").run_fail().assert_stderr_contains("struct literal body without path");
```

Comment:
Suggestion: the `RUSTC_TRANSLATION_NO_DEBUG_ASSERT=1` env var might be load-bearing, see https://github.com/rust-lang/rust/blob/69e36d65f93f81f4cbd8a3191c33be3fc201f095/compiler/rustc_errors/src/translation.rs#L109-L118

We probably want to have a `Rustc` factory fn that presets that env var:

```rs
fn rustc_no_translation_assert() -> Rustc {
    rustc().env("RUSTC_TRANSLATION_NO_DEBUG_ASSERT", "1")
}
```

Reply:
@rustbot author

Reply:
`rustc()` returns a temporary `&mut Rustc` which cannot be returned at first glance (unless there is some lifetime magic to pull it off).

I added the environment variable to the rustc calls in the test manually, though in my local tests it seemed to have no effect. Then again, this might be due to me using stage 0 compilation.

Reply:
Oh right, my bad. Hm.